.circleci/config.yml

```diff
@@ -5,37 +5,36 @@ orbs:
 jobs:
   build:
     docker:
-      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester-v4
     steps:
       - checkout
       - run:
           name: 'Setup virtual env'
           command: |
-            virtualenv -p python3 /usr/local/share/virtualenvs/tap-salesforce
+            python3 -mvenv /usr/local/share/virtualenvs/tap-salesforce
             source /usr/local/share/virtualenvs/tap-salesforce/bin/activate
-            pip install .
-            pip install pylint
-            pylint tap_salesforce -d missing-docstring,invalid-name,line-too-long,too-many-locals,too-few-public-methods,fixme,stop-iteration-return,no-else-return,chained-comparison,broad-except,no-else-raise
+            pip install -U 'pip<19.2' 'setuptools<51.0.0'
+            pip install .[dev]
       - run:
-          name: 'Unit Tests'
+          name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-salesforce/bin/activate
-            pip install nose
-            nosetests
```

Comment:
Are we discarding the old unit tests? Can we keep a step for them if they existed?

Reply:
They do not exist, it had been running nothing before and passing. See unittests https://app.circleci.com/pipelines/github/singer-io/tap-salesforce/307/workflows/47ddaee4-a2f1-4b82-b986-945de81077a1/jobs/744

Reply:
Lol. Yeah then remove it.